server/src/main/java/org/candlepin/audit/EventBuilder.java

```diff
@@ -42,23 +42,40 @@
 
     private static final Logger log = LoggerFactory.getLogger(EventBuilder.class);
 
-    private final EventFactory factory;
     private final ObjectMapper mapper;
 
     private Event event;
 
     public EventBuilder(EventFactory factory, Target target, Type type) {
-        this.factory = factory;
         this.mapper = new ObjectMapper();
 
         event = new Event(type, target, null, factory.principalProvider.get(),
-                null, null, null, null, null, null, null);
+                null, null, null, null, null, null);
     }
 
-    /*  Implementation note (2017/10/16):
-        This method is currently used instead of setOldEntity, since oldEntity was removed,
-        and it will also be used in the future instead of setNewEntity once newEntity is removed.
-        This is part of simplifying the data Events hold.
+    /**
+     * Utility method used to pass in the old and new version of the modified
+     * entity for {@link Type#MODIFIED} events.
+     * @param oldEntity The target entity before modification
+     * @param newEntity The target entity after modification
+     * @return The builder object
+     */
+    public EventBuilder setEventData(Eventful oldEntity, Eventful newEntity) {
+        if (!event.getType().equals(Type.MODIFIED)) {
+            throw new IseException("This method is only for type MODIFIED Events.");
+        }
+        setEventData(oldEntity);
+        return setEventData(newEntity);
+    }
+
+    /**
+     * This method is used with any type of event and any target entity.
+     * <br><br/>
```

Comment:
Try to be consistent with your markup here. Also, using ```<p></p>``` will give you a more consistent paragraph-style break than relying on line height from line breaks.